site/assets/js/search.js

```diff
@@ -9,58 +9,122 @@
     return
   }
 
-  var inputElement = document.getElementById('search-input')
-  var siteDocsVersion = inputElement.getAttribute('data-docs-version')
-
-  document.addEventListener('keydown', function (event) {
-    if (event.ctrlKey && event.key === '/') {
-      event.preventDefault()
-      inputElement.focus()
-    }
-  })
+  function isIconHit(hit) {
+    return hit.hierarchy.lvl0 === 'Bootstrap Icons'
+  }
 
-  function getOrigin() {
-    var location = window.location
-    var origin = location.origin
+  function IconName(hit) {
+    return hit.url
+      .replace(/https:\/\/icons.getbootstrap.com\/icons\//g, '')
+      .replace(/\/#content/g, '')
+  }
 
-    if (!origin) {
-      var port = location.port ? ':' + location.port : ''
+  function IconSVG(hit) {
+    return {
+      type: 'svg',
+      ref: undefined,
+      constructor: undefined,
+      key: undefined,
+      props: {
+        width: 20,
+        height: 20,
+        fill: 'currentColor',
+        children: {
+          type: 'use',
+          ref: undefined,
+          constructor: undefined,
+          key: undefined,
+          props: {
+            xlinkHref:
+              '/docs/5.0/assets/img/bootstrap-icons.svg#' + IconName(hit),
+          },
+        },
+      },
+    }
+  }
 
-      origin = location.protocol + '//' + location.hostname + port
+  function Hits({ hit, children }) {
```

Comment:
based on the lint error, I think destructuring might not be allowed in this file